lib/symbioticpy/symbiotic/options.py

```diff
@@ -200,6 +200,171 @@ def parse_command_line():
+                            help='Use 32-bit environment')
+    arg_parser.add_argument('--64', dest='use_64bit_env', action='store_true', default=True,
+                            help='Use 64-bit environment')
+    arg_parser.add_argument('--timeout', type=int, default=None, help='Set timeout to t seconds')
+    arg_parser.add_argument('--repeat-slicing', type=int, help='Repeat slicing N times')
+    arg_parser.add_argument('--instrumentation-timeout', type=int, metavar='N',
+                            help=('Set timeout for instrumentation (if instrumentation\n'
+                                  'timeouts, the original bitcode is used and slicing\n'
+                                  'is skipped)'))
+    arg_parser.add_argument('--executable-witness', action='store_true', default=False,
+                            help='Produce an executable witness')
+    arg_parser.add_argument('--explicit-symbolic', action='store_true', default=False,
+                            help='Do not make all memory symbolic, but rely on calls to __VERIFIER_nondet_*')
+    arg_parser.add_argument('--no-slice', action='store_true', default=False, help='Do not slice the code')
+    arg_parser.add_argument('--no-optimize', action='store_true', default=False,
+                            help='Don\'t optimize the code (same as --optimize=none)')
+    arg_parser.add_argument('--no-instrument', action='store_true', default=False,
+                            help='Don\'t instrument the code, for debugging.')
+    arg_parser.add_argument('--no-verification', action='store_true', default=False,
+                            help='Do not run verification phase (handy for debugging)')
+    arg_parser.add_argument('--no-replay-error', action='store_true', default=False,
+                            help='Do not replay a found error on non-sliced code (overrides --sv-comp)')
+    arg_parser.add_argument('--no-integrity-check', action='store_true', default=False,
+                            help='Does not run integrity check. For development only.')
+    arg_parser.add_argument('--no-link-undefined', action='store_true', default=False,
+                            help='Do not to find and link undefined functions.')
+    arg_parser.add_argument('--dump-env', action='store_true',
+                            default=False, help='Only dump environment variables (for debugging)')
+    arg_parser.add_argument('--dump-env-cmd', action='store_true',
+                            default=False, help='Dump environment variables for using them in command line')
+
+    # TODO(MichalHe): @review maybe it should be more robust to use choices here
+    arg_parser.add_argument('--no-link', metavar='CATEGORIES',
+                            help=('Do not link missing functions from the given category\n'
+                                  '(libc, svcomp, verifier, posix, kernel). The argument\n'
+                                  'is a comma-separated list of values.'))
```

Comment:
Yeah, categories would be awesome! The list is the following (without `symbitoicpy`): https://github.com/staticafi/symbiotic/tree/main/lib